gradle.properties

```diff
@@ -5,7 +5,7 @@ VERSION=0.50.37
 
 org.gradle.parallel=true
 
-org.gradle.jvmargs=-Xmx8g -Xss4m
+org.gradle.jvmargs=-Xmx12g -Xss4m
```

Comment:
This might be dangerous as our CI runners have 16GB and may run multiple gradle daemons concurrently. Then again, if it was a problem, you would have noticed by now.

Reply:
i'll see if I can just revert this